programs/mango-v4/src/instructions/group_edit.rs

```diff
@@ -106,5 +108,24 @@ pub fn group_edit(
         group.buyback_fees_expiry_interval = buyback_fees_expiry_interval;
     }
 
+    if let Some(fees_bps) = token_conditional_swap_taker_fee_bps_opt {
+        msg!(
+            "Token conditional swap taker fee bps old {:?}, new {:?}",
+            group.token_conditional_swap_taker_fee_bps,
+            fees_bps
+        );
+        require_gte!(fees_bps, 0); // values <0 are not currently supported
+        group.token_conditional_swap_taker_fee_bps = fees_bps;
+    }
+    if let Some(fees_bps) = token_conditional_swap_maker_fee_bps_opt {
+        msg!(
+            "Token conditional swap maker fee bps old {:?}, new {:?}",
+            group.token_conditional_swap_maker_fee_bps,
+            fees_bps
+        );
+        require_gte!(fees_bps, 0); // values <0 are not currently supported
```

Comment:
hmm why dont we want to support -ve maker fees?

Reply:
It's just something I haven't thought about yet. Even though these makers are technically providing liquidity, it's not really a cohesive market. Also, we already need to incentivize the takers to execute these, so it feels odd to shift funds from takers to makers somehow.